kubernetes/samples/scripts/create-rcu-credentials/README.md

```diff
@@ -23,9 +23,9 @@ The parameters are as follows:
 
 ```  
   -u username for schema owner (regular user), must be specified.
-  -p password for schema owner (regular user), must be specified.
+  -p password for schema owner (regular user), must be provided through -p argument or user will be prompted to enter a value.
```

Comment:
provided through -> provided using the ("using" is the better verb to use in this phrase; please make this correction globally. Thank you.)